ReactNative/ReactTwitter/core/views/debug/debug-screen.js

```diff
@@ -14,13 +14,15 @@ var PushNotificationsScreenView = require('./push-notifications-screen.js')
 var OauthView = require('./oauth-screen.js')
 var DebugAnimationScene = require('./debug-animation-scene.js')
 var DebugNativeComponentScene = require('./debug-native-component-scene.js')
+var DebugUITestingScene = require('./debug-uitesting-ios.js')
```

Comment:
It would be nice to rename it `debug-uitesting`, in the hope that we'll be able to do the same with Android in the future